TESTING/EIG/dchkec.f

```diff
@@ -189,9 +189,15 @@ SUBROUTINE DCHKEC( THRESH, TSTERR, NIN, NOUT )
          OK = .FALSE.
          WRITE( NOUT, FMT = 9991 )RLAQTR, LLAQTR, NLAQTR, KLAQTR
       END IF
+*
+      CALL DGET40( RTGEXC, LTGEXC, NTGEXC, KTGEXC, NIN )
+      IF( RLAQTR.GT.THRESH ) THEN
```

Comment:
Hi @thijssteel. Does my suggestion make sense? Thanks.
```suggestion
      IF( RTGEXC.GT.THRESH ) THEN
```

Reply:
yes, very good catch. I see i made the same mistake in the single precision version.